geowebcache/core/pom.xml

```diff
@@ -199,7 +234,6 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Interesting, what's up here? I guess this groupid is a default, when talking about maven plugins?